src/schemas/json/mtad.json

```diff
@@ -306,7 +306,7 @@
                     "type": "boolean"
                 },
                 "dependency-type": {
-                    "description": "Defines the memory allocated to the application.",
+                    "description": "Defines behavior on resolving deployment order with cyclic dependencies ",
```

Comment:
I'd add an example here:
```
Influences deployment order whenever cyclic dependencies exist. For example, if module A requires module B, and vice versa, but module A has a dependency-type parameter set to "hard", then it would be deployed prior to module B.
```